app/services/fees/calculator.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
Have you considered calculating everything in tinybar and wei, thus skipping the usage of `decimals`?

Reply:
This seemed to be the most straight-forward way in the time of doing this task to me. I thought that we would lose precision of calculations if we did it the suggested way.

Reply:
That is a good suggestion. We won't lose precision. I think that we will have fewer conversions as everything will be done in the lowest denomination and only one at the end in which we will convert the HBAR/ETH exchange rate to TINYBAR/WEI, which would be basically:
`exchangeRate*(10**10)`